.env.local.php.example

```diff
@@ -22,4 +22,6 @@ return [
     ],
     // The tag PHID that indicates whether a task is in review
     'REVIEW_TAG_PHID' => '',
+    // comma separated names of Phabricator user names that are allowed to create projects on Phragile
+    'PHRAGILE_ADMINS' => ''
```

Comment:
Does Laravel not provide a more integrated way of doing this? And if so, why did you pick this approach? Just curious, not saying there is anything bad here.